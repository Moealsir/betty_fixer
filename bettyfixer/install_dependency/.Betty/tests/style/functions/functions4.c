void func0(int var)
{
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	/* 10 */
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	/* 20 */
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	/* 30 */
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	/* 40 */
}

void func1(int var)
{
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	/* 10 */
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	/* 20 */
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	/* 30 */
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	++var;
	/* 40 */
	++var;
}
