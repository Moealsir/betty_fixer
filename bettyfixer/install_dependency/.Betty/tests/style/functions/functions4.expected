tests/style/functions/functions4.c:87: WARNING: More than 40 lines in a function
total: 0 errors, 1 warnings, 88 lines checked
